nextcord/enums.py

```diff
@@ -716,6 +716,11 @@ class AutoModerationActionType(IntEnum):
     timeout = 3
 
 
+class SortOrderType(IntEnum):
```

Comment:
`__all__` needs to be modified so it is exposed via the wildcard imports in `__init__.py`